SteamBot/Bot.cs

```diff
@@ -336,6 +364,24 @@ public void HandleBotCommand(string command)
             }
         }
 
+        public void HandleInput(string input)
+        {
+            consoleInput = input;
+        }
+
+        public string WaitForInput()
+        {
+            consoleInput = null;
+            while (true)
+            {
+                if (consoleInput != null)
+                {
+                    return consoleInput;
+                }
+                Thread.Sleep(5);
+            }
+        }
```

Comment:
Hey, donchu bring me into this @waylaidwanderer . I wrote it that way because I needed to get rid of the winform dependency in half an hour and I think we can agree sir that a GUI is worse than a sleeping thread that can't do anything useful anyway :(

Then again, I guess that's a perfectly valid argument for you too. Hm. Still though it's good people are paying attention to what really matters about this PR.